api/models/event.go

```diff
@@ -26,9 +27,13 @@ func (e *CreateEvent) Validate() error {
 }
 
 type DynamicEvent struct {
-	Endpoint     DynamicEndpoint     `json:"endpoint"`
-	Subscription DynamicSubscription `json:"subscription"`
-	Event        DynamicEventStub    `json:"event"`
+	URL            string          `json:"url" valid:"required~please provide a url"`
+	Secret         string          `json:"secret" valid:"required~please provide a secret"`
+	EventTypes     []string        `json:"event_types"`
+	Data           json.RawMessage `json:"data" valid:"required~please provide your data"`
+	ProjectID      string          `json:"project_id" swaggerignore:"true"`
```

Comment:
@danvixent, Why do we need the `ProjectID` in the payload? The API Key already holds the context.